docs/http_client.md

```diff
@@ -6,6 +6,100 @@ For example, if you need to connect to a CRUD, you have to use a Proxy towards t
 
 You can get a proxy calling these methods both on `Request`(the first argument of handler) and `Service` (the Fastify instance):
 
+* `getHttpClient(baseURL, options)` - returns an http client configured with the given base URL and options.
+  * `baseURL` - the base URL of the service, with protocol, host and it is possible to add a base prefix. The prefix must ends with a slash. Keep in mind that in method functions you must start the path without slash if you want to reuse the path prefix.
+  * `options` - an object with the following optional fields:
+    * `headers` - an object that represents the set of headers to send to the service
+    * `timeout` - set a request timeout
+    * `cert` - set a custom certificate to use for the requests
+    * `key` - set a custom key to use for the requests
+    * `ca` - set a custom ca to use for the requests
+    * `logger` - the Pino logger instance, it is used to log request (headers, payload and url) and response (headers, payload and status code) in trace level and error message if there is an error during the API call. If not passed, no log are printed. Keep in mind that headers, payload and url could contains sensitive information. If it is the case, do not pass the logger instance or use the redact options to hide the sensitive information ([read here](https://docs.mia-platform.eu/docs/runtime_suite_libraries/lc39/service-options) for more information).
+
+Potentially, the `getDirectServiceProxy` method allows you to also query services outside the Platform. In this case, however, it is necessary to bear in mind that the platform headers will be automatically forwarded.
```

Comment:
Why is the `getDirectServiceProxy` cited here? aren't you describing the `getHttpClient` function?